dotnet/src/InternalUtilities/samples/InternalUtilities/BaseTest.cs

```diff
@@ -59,7 +60,7 @@ protected Kernel CreateKernelWithChatCompletion()
         return builder.Build();
     }
 
-    protected BaseTest(ITestOutputHelper output)
+    protected BaseTest(ITestOutputHelper output, bool redirectSystemConsoleOutput = false)
```

Comment:
I'd be tempted to make the default `true`...I'm not sure any samples are using System.Console anyway.

Reply:
I think I'll keep it opt-in for now, but if this is turns out to be a common issue then we can change it and make sure there are no side effects.